src/main/java/fr/insalyon/creatis/moteurlite/MoteurLite.java

```diff
@@ -0,0 +1,76 @@
+package fr.insalyon.creatis.moteurlite;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import fr.insalyon.creatis.gasw.Gasw;
+import fr.insalyon.creatis.moteurlite.boutiquesParser.ParseBoutiquesFile;
+import fr.insalyon.creatis.moteurlite.inputsParser.ParseInputsFile;
+import fr.insalyon.creatis.moteurlite.iterationStrategy.IterationStrategy;
+
+/**
+ * 
+ * @author Sandesh Patil [https://github.com/sandepat]
+ * 
+ */
+
+public class MoteurLite {
+    private Gasw gasw = Gasw.getInstance();
+    private int sizeOfInputs;
+    private String bashScript;
+    IterationStrategy iterationStrategy = new IterationStrategy();
+    Workflowsdb workflowsdb = new Workflowsdb();
+
+    public static void main(String[] args) throws Exception {
+        new MoteurLite(args);
+    }
+
+    public MoteurLite(String[] args) throws Exception {
+        String workflowId = args[0];
+        String boutiquesFilePath = args[1];
+        String inputsFilePath = args[2];
+
+        //load bash script
+        bashScript = ScriptLoader.loadBashScript();
+
+        //parse boutiques file
+        ParseBoutiquesFile parseBoutiquesFile = new ParseBoutiquesFile(boutiquesFilePath);
```

Comment:
Changes Made
    Class Renaming: The class ParseBoutiquesFile is renamed to BoutiquesService to reflect its role more appropriately.

    Method Creation: The logic for parsing the file is moved into a method called parseFile.

    Field Initialization: The fields are initialized within the parseFile method instead of the constructor.

    Getter Methods: Provided getter methods for accessing the parsed data.

    Constructor: An empty default constructor is added.